core/src/main/scala/org/http4s/syntax/KleisliSyntax.scala

```diff
@@ -4,13 +4,22 @@ package syntax
 import cats._
 import cats.data.{Kleisli, OptionT}
 
-trait KleisliResponseSyntax {
-  implicit def http4sServiceSyntax[F[_]: Functor, A](
+trait KleisliSyntax {
+  implicit def http4sKleisliResponseSyntax[F[_]: Functor, A](
       service: Kleisli[OptionT[F, ?], A, Response[F]]): KleisliResponseOps[F, A] =
     new KleisliResponseOps[F, A](service)
+
+  implicit def http4sKleisliSYntax[F[_]: Functor, A, B](
```

Comment:
Weird Capitalization here. Although I thought we might be removing it.

Reply:
Yes, I'd like to see whether Cats is interested in providing this.  `KleisliResponseOps` is our jam, but this one has nothing to do with HTTP.

Reply:
Capitalization was a typo.

I don't disagree that this should be provided somewhere else, but having it in a released version of cats will not happen overnight.

Inlining the functionality is basically `.mapF(OptionT.liftF(_))`. Would that be an acceptable solution?